scss/_variables.scss

```diff
@@ -1,3 +1,3 @@
-$govuk-assets-path: "/restricted-word/public/";
+$govuk-assets-path: "/admin/restricted-word/public/";
```

Comment:
Why are you adding admin in the path for this change?

Reply:
because it is incorrect path and in browser show a lot of errors

gupl copy to 
```
        .src(["./node_modules/govuk-frontend/govuk/assets/**/*"])
        .pipe(gulp.dest("./dist"));
});

in app.ts dist pointed as `/public` folder
```
app.use(`/${config.urlPrefix}/public`, express.static(path.join(__dirname, "../dist")));
```

config.ts
```
    urlPrefix: "admin/restricted-word",
 ```